src/functions/L1plusL2.jl

```diff
@@ -0,0 +1,56 @@
+# (weighted) sum of L2 norm and L1 norm
+# for Group Lasso, this can be used together with src/calculus/slicedSeparableSum
+
+using ProximalOperators
+import ProximalOperators: prox!
```

Comment:
using and import would only be needed of you try to run this file separately from the package, which might have been the case when you tested it. But it is not needed since the code already "lives" in the module now.

Reply:
I am not so experienced with writing Julia packages. So I can always use any functions from inside the module without any import statement (except maybe for the testing)?

Reply:
Yes, when you are inside the module (which you remain in when including the file in the module), the functions are all in that "namespace" already. The tests are not in the same module, so there it is needed to import.